common/api/core-geometry.api.md

```diff
@@ -5349,6 +5399,8 @@ export class SineCosinePolynomial {
 // @public
 export class SmallSystem {
     static eliminateFromPivot(rowA: Float64Array, pivotIndex: number, rowB: Float64Array, a: number): boolean;
+    // (undocumented)
```

Comment:
Is this APi intentionally `@public`? Public and beta APIs need documentation.

Reply:
Yes, it was an oversight. Doc added.